src/Components/Articles.jsx

```diff
@@ -0,0 +1,44 @@
+import { Route, Routes, Link } from "react-router-dom";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+function Articles() {
+  const [allArticles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://nc-news-be-project-lndv.onrender.com/api/articles")
+      .then((response) => {
+        return response.data;
+      })
+      .then((body) => {
+        setArticles(body.articles);
+      });
+  });
+  return (
```

Comment:
Could this be compacted into another component? Perhaps an ArticleCard component